README.rst

```diff
@@ -24,6 +24,20 @@ Motr
 At the core of `CORTX <https://github.com/Seagate/cortx>`_ lies Motr.  Motr is a distributed object storage system, targeting `mass capacity storage <https://www.seagate.com/products/storage/object-storage-software/>`_
 configurations. To ensure the most efficient storage utilization, Motr interacts directly with block devices (i.e. it does not _layer_ on a local file system).  The Motr design was heavily influenced by the Lustre file system, NFSv4 and database technology. It must be noted that traditional file system properties (hierarchical directory namespace, strong POSIX consistency guarantees, &c.) are no longer desirable or achievable at mass capacity. Instead, Motr is a more general storage system that provides an optional file system interface. This allows wider range of deployments, including cloud.
 
+Following are the features of the Motr submodule:
```

Comment:
Please remove the word "submodule".

At first glimpse, I was confused it with the "git submodule".

Can we say: 
* Cortx Motr
* The Motr subsystem
* The Motr storage system
* The Motr object storage system
* The Motr
?

Reply:
CORTX Motr is better.